WTWSMS/history/provinces/478 - Monemvasia.txt

```diff
@@ -4,13 +4,13 @@
 title = c_monemvasia
 
 # Settlements
-max_settlements = 4
+max_settlements = 6
```

Comment:
Prosperity doesn't add new holding slots. Changing that line allows for six empty holding slots, as opposed to three. It was intended to prevent the City of Sparta from not being able to be added, but I realized that you can force a new holding slot, so I'll change that now.